mettle/api/api.go

```diff
@@ -393,17 +396,28 @@ func (s *server) stopStream(digest *pb.Digest, ch <-chan *longrunning.Operation)
 
 // Receive runs forever, receiving responses from the queue.
 func (s *server) Receive() {
-	for {
-		msg, err := s.responses.Receive(context.Background())
-		if err != nil {
-			log.Fatalf("Failed to receive message: %s", err)
-		}
-		s.process(msg)
-		msg.Ack()
+	wg := sync.WaitGroup{}
+	// yes, <= because we're guaranteeing that there's at least one worker if pollers is 0
+	for i := 0; i <= s.numPollers; i++ {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			for {
+				msg, err := s.responses.Receive(context.Background())
+				if err != nil {
+					log.Errorf("Failed to receive message: %s", err)
+					return
```

Comment:
Yeah, not sure if we should be treating them as completely bulk-headed or not.

Could fatal immediately rather than allow any work, to keep existing semantics the same, does mean we lose a bit of "real" work, but we've set the retry timeout to be small anyway.